CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in the repo. Unless a
+# later match takes precedence. Owners will be requested for
+# review when someone opens a pull request.
+*    @canonical/soleng-reviewers
+*    @canonical/data-ai-kubeflow
```

Comment:
suggestion(blocking): Let's also add `*       @canonical/kubeflow`

That's what we also use in our repos
https://github.com/canonical/bundle-kubeflow/blob/main/CODEOWNERS

Reply:
Addressed in https://github.com/canonical/velero-operator/commit/71a403a4e5ec2f5e639ef27356edc5e6410c0a23

Resolving!